lib/fastboot-response.js

```diff
@@ -0,0 +1,20 @@
+var cookie = require('cookie');
+var FastBootHeaders = require('./fastboot-headers');
+
+function FastbootResponse(response) {
+  this.headers = new FastBootHeaders(response.headers);
+  this.statusCode = 200;
+}
+
+FastbootResponse.prototype.cookie = function (cookieString) {
```

Comment:
Do we need to do this? Or is this something that `ember-cookies` can now handle?